test/e2e/config/k3s-docker.yaml

```diff
@@ -0,0 +1,96 @@
+---
+# E2E test scenario using local dev images and manifests built from the source tree for following providers:
+# - cluster-api
+# - bootstrap k3s
+# - control-plane k3s
+# - docker
+managementClusterName: k3s-e2e
```

Comment:
Set fixed name for mgmt cluster may cause problem when running e2e test concurrently. Let's discover if we can let test framework to generate name dynamicly.